### ISSUE TITLE
Increase playlist button size

### ISSUE THREAD

**Issue by nikclayton** [nikclayton/android-squeezer#112]
```
Hi,

instead of having a small button on playlist items (for tasks like play, download,
move, etc.) with a size even not really recognizable it would be IMHO much better to
use the whole playlist entry as button.
Therefore it would not happen anymore that a playlist entry accidentally is played,
when the button is missed.
```

Original issue reported on code.google.com by `thenktor` on 2015-01-12 17:21:58

**Comment by matt-matt2** [comment]
Plus, I've seen instances on some screen sizes (Nexus 9 for one) where the button is actually partially hidden behind the scroll bar. However I can see that currently the button provides a context menu, whilst the click on the item itself is used as a direct action, so I'd rather keep that capability but make the small button a bit bigger! :)

**Comment by matt-matt2** [comment]
I would like a quick action, such as a swipe, to remove items from the playlist, however.

**Comment by kaaholst** [comment]
This is fixed in Squeezer version 2.0.0 and newer, closing.
